Cargo.toml

```diff
@@ -7,7 +7,7 @@ resolver = "2"
 rust-version = "1.75"
 
 [dependencies]
-micro-rdk = {git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32"], version = "0.1.9-rc3", rev = "4f0814a"}
```

Comment:
this is autogenerated when I choose the sensor type from the module project cli, wanna fix it there?